doc/users/whats_new.rst

```diff
@@ -10,251 +10,325 @@ revision, see the :ref:`github-stats`.
+were previously displayed as ``1e+04``.
 
-    fig = plt.figure()
-    gs0 = fig.add_gridspec(3, 1)
-    ax1 = fig.add_subplot(gs0[0])
-    ax2 = fig.add_subplot(gs0[1])
-    gssub = gs0[2].subgridspec(1, 3)
-    for i in range(3):
-        fig.add_subplot(gssub[0, i])
+MouseEvent button attribute is now an IntEnum
+----------------------------------------------
+The :attr:`button` attribute of `-.MouseEvent` instances can take the values
+None, 1 (left button), 2 (middle button), 3 (right button), "up" (scroll), and
+"down" (scroll).  For better legibility, the 1, 2, and 3 values are now
+represented using the `IntEnum` class `matplotlib.backend_bases.MouseButton`,
+with the values `MouseButton.LEFT` (``== 1``), `MouseButton.MIDDLE` (``== 2``),
+and `MouseButton.RIGHT` (``== 3``).
 
 
-Figure has an `~.figure.Figure.add_artist` method
--------------------------------------------------
+Configuration, Install, and Development
+=======================================
+
+The MATPLOTLIBRC environment variable can now point to any "file" path
+-----------------------------------------------------------------------
+This includes device files; in particular, on Unix systems, one can set
+``MATPLOTLIBRC`` to ``/dev/null`` to ignore the user's matplotlibrc file and
+fall back to Matplotlib's defaults.
 
-A method `~.figure.Figure.add_artist` has been added to the
-:class:`~.figure.Figure` class, which allows artists to be added directly
-to a figure. E.g. ::
+As a reminder, if ``MATPLOTLIBRC`` points to a directory, Matplotlib will try
+to load the matplotlibrc file from ``$MATPLOTLIBRC/matplotlibrc``.
 
-   circ = plt.Circle((.7, .5), .05)
-   fig.add_artist(circ)
 
-In case the added artist has no transform set previously, it will be set to
-the figure transform (``fig.transFigure``).
-This new method may be useful for adding artists to figures without axes or to
-easily position static elements in figure coordinates.
+Allow LaTeX code ``pgf.preamble`` and ``text.latex.preamble`` in MATPLOTLIBRC file
+----------------------------------------------------------------------------------
+Previously, the rc file keys ``pgf.preamble`` and ``text.latex.preamble`` were parsed using commmas as separators. This would break valid LaTeX code, such as::
 
+\usepackage[protrusion=true, expansion=false]{microtype}
 
+The parsing has been modified to pass the complete line to the LaTeX system,
+keeping all commas. Passing a list of strings from within a Python script still works as it used to.
 
-``:math:`` directive renamed to ``:mathmpl:``
----------------------------------------------
 
-The ``:math:`` rst role provided by `matplotlib.sphinxext.mathmpl` has been
-renamed to ``:mathmpl:`` to avoid conflicting with the ``:math:`` role that
-Sphinx 1.8 provides by default.  (``:mathmpl:`` uses Matplotlib to render math
-expressions to images embedded in html, whereas Sphinx uses MathJax.)
+Added support for MacOSX backend for PyPy
```

Comment:
This is actually a bigger deal, and should probably be higher up:

Matplotlib no longer requires framework app build on MacOSX backend
-----------------------------------------------------------------------

Previous versions of matplotlin required a Framework build of python to
work.  The app type was updated to no longer require this, so the MacOSX
backend should work with non-framework python.

Reply:
moved this up.